aio_pika/patterns/base.py

```diff
@@ -34,7 +34,21 @@ class Base:
     CONTENT_TYPE = "application/python-pickle"
 
     def serialize(self, data: Any) -> bytes:
+        """ Serialize data to the bytes.
+        Uses `pickle` by default.
+        You should overlap this method when you want to change serializer
+
+        :param data: Data which will be serialized
+        :returns: bytes
```

Comment:
Is specification of returned types is necessary within doctstring? There is already typing exists.

Reply:
this docstrings just moved.